src/lang/hr-HR.json

```diff
@@ -348,7 +348,7 @@
     "Discard": "Odbaci",
     "Cancel": "Otkaži",
     "Powered by": "Pokreće",
-    "Saved": "Spremljeno",
+    "Saved.": "Spremljeno.",
```

Comment:
We are inconsistent between `Saved.` and `Saved`. 
Please unify this to one message or add both keys to the translations

Reply:
Fixed.